modules/cnv-vm-storage-volume-types.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// * cnv_users_guide/cnv-create-vms.adoc
+
+[id="cnv-vm-storage-volume-types_{context}"]
+= Types of storage volumes for virtual machines
+
+[horizontal]
+*ephemeral*::
+A local copy-on-write (COW) image that uses a network volume as a
+read-only backing store. The backing volume
+must be a *PersistentVolumeClaim*. The ephemeral image is created when
+the virtual machine starts, and stores all writes locally. The ephemeral
+image is discarded when the virtual machine is stopped, restarted, or
+deleted. The backing volume (PVC) is not mutated in any way.
+
+*persistentVolumeClaim*::
+Attaches an available PV to a virtual machine. This allows for the
+virtual machine data to persist between sessions.
++
+Importing an existing virtual machine disk into a PVC using
```

Comment:
s/using/by using

Reply:
Changed to:

Importing an existing virtual machine disk into a PVC by using